monai/transforms/utility/dictionary.py

```diff
@@ -1003,7 +1003,8 @@ def __call__(self, data: Mapping[Hashable, NdarrayOrTensor]) -> Dict[Hashable, N
             ret = self._transform(data=d[key], func=func)
             if overwrite:
                 d[key] = ret
-            self.push_transform(d, key)
+            if self.inv_func is not None:
```

Comment:
Hi @rijobro ,

As the type-hints stated, the input data type of `inv_func` is `Callable` or `Sequence[Callable]`, so maybe we don't need to check the `None` case here? I see we usually don't check unexpected python data types so far..

And BTW, even checking it, `self.inv_func = ensure_tuple_rep(inv_func, len(self.keys))` makes `self.inv_func` to be a tuple of `None`, so `if self.inv_func is not None` will be False, should check every item of the tuple.
What do you think?

Thanks.